cmd/fireaptos/cli/firehose.go

```diff
@@ -103,7 +103,7 @@ func init() {
 			}
 
 			return firehoseApp.New(appLogger, &firehoseApp.Config{
-				OneBlocksStoreURL:       MustReplaceDataDir(sfDataDir, viper.GetString("common-one-blocks-store-url")),
+				OneBlocksStoreURL:       MustReplaceDataDir(sfDataDir, viper.GetString("common-one-block-store-url")),
 				MergedBlocksStoreURL:    MustReplaceDataDir(sfDataDir, viper.GetString("common-merged-blocks-store-url")),
 				BlockStreamAddr:         viper.GetString("common-relayer-addr"),
```

Comment:
You seem to be missing a search/replace of`common-relayer-addr` (should now become `common-live-blocks-addr`)